stackit/internal/services/objectstorage/credential/resource_test.go

```diff
@@ -32,7 +31,8 @@ func (c *objectStorageClientMocked) CreateProjectExecute(_ context.Context, proj
 }
 
 func TestMapFields(t *testing.T) {
```

Comment:
Does it make sense to add a test case for the "Harmonize the timestamp format"?

Reply:
Yes it does